video/out/vo_wlshm.c

```diff
@@ -209,6 +209,8 @@ static int resize(struct vo *vo)
     vo->target_params = &p->sws->dst;
     mp_mutex_unlock(&vo->params_mutex);
 
+    vo_wayland_handle_color(wl);
```

Comment:
I think `vo_wlshm.c`  should also set target_params from input frame, no?

Reply:
Yeah I think this just got mixed around from rebasing from previous changes. Should this just copy what `vo_dmabuf_wayland` does?

Reply:
Yes, I think so.

Reply:
I looked at `vo_wlshm` again and actually it needs a ton of work before it can even be usable. It's hardcoded to use only bgr0 or 0rgb depending on the endian so there's really zero use in exposing any of this color stuff until that gets smarter. I will just drop this to only `vo_dmabuf_wayland`. We can always add more VOs later when it makes sense.